<?xml version="1.0"?>
<launch>

  <include file="$(find gazebo_ros)/launch/empty_world.launch">
    <arg name="world_name" value="$(find legibot)/worlds/restaurant.world"/>
    <arg name="paused" value="false"/>
    <arg name="verbose" value="true"/>
  </include>

  <include file="$(find legibot)/launch/spawn_pepper.launch" >
    <arg name="x" value="-3.0"/>
    <arg name="y" value="3.0"/>
    <arg name="yaw" value="-1.57"/>
  </include>

  <include file="$(find legibot)/launch/spawn_observer.launch">
    <arg name="x" value="-3.7"/>
    <arg name="y" value="-8.5"/>
    <arg name="yaw" value="2.3"/>
    <arg name="obj_name" value="observer1" />
  </include>

<!--   <include file="$(find legibot)/launch/spawn_observer.launch"> -->
<!--     <arg name="x" value="-1.3"/> -->
<!--     <arg name="y" value="-8"/> -->
<!--     <arg name="yaw" value="1.8"/> -->
<!--     <arg name="obj_name" value="observer2" /> -->
<!--   </include> -->

  <include file="$(find teleop_twist_joy)/launch/teleop.launch" />

  <include file="$(find legibot)/launch/robot_controller.launch" >
<!--     <arg name="x" value="-2.6"/> -->
<!--     <arg name="y" value="-7.12"/> -->
<!--     <arg name="yaw" value="0.0"/> -->
  </include>

<!--   <node name="robot_controller" pkg="legibot" type="robot_controller.py" output="screen"> -->
<!--     <param name="goal" value="(-2.6,-7.12)"/> -->
<!--   </node> -->

<!--   <node name="observer_recorder" pkg="legibot" type="record_observer.py" output="screen" /> -->
<!--   <node name="legibility" pkg="legibot" type="opf_based_legibility.py" output="screen" /> -->

</launch>
